x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/UserToken.java

```diff
@@ -110,7 +110,7 @@ public String getId() {
     /**
      * The version of the node this token was created on
      */
-    Version getVersion() {
+    TransportVersion getTransportVersion() {
```

Comment:
From the comment, this is the actual node version. Do we want to change this to TransportVersion?

Reply:
I updated the comment. I think this should be a transportVersion as per https://github.com/elastic/elasticsearch/pull/93089/files#r1087901056